config/log_rules.json

```diff
@@ -11,7 +11,7 @@
       "fail": {
         "active": true,
         "start_pattern": "^(?<fail>Failed to .+)\\n",
-        "end_pattern": "Filename: (?:[\\w/:]+/(?<file>\\w+\\.\\w+))? Line: (?<line>-?\\d+)",
```

Comment:
are we too restrictive? What about other missing characters?

Reply:
We could add "-", "_", and "\s" as they are common in filenames. WDYT?

Reply:
Wonder if we just should relax the whole matching expression between Filename and Line.

Reply:
If not possible then you last patch is better

Reply:
Relaxing shouldn't cause much trouble, and rather ensure that rules do terminate which is rather a good thing, so I'll do that.